client-ui/src/app/model/types/operationType.ts

```diff
@@ -25,19 +25,21 @@ import { GenericComponent } from 'src/app/components/lineage/lineage-details/ope
 import { AggregateComponent } from 'src/app/components/lineage/lineage-details/operation-properties-details/aggregate/aggregate.component';
 import { WriteComponent } from 'src/app/components/lineage/lineage-details/operation-properties-details/write/write.component';
 import { AliasComponent } from 'src/app/components/lineage/lineage-details/operation-properties-details/alias/alias.component';
+import { PropertyErrorComponent } from 'src/app/components/lineage/lineage-details/operation-properties-details/property-error/property-error.component';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [' should be "](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4600414)